cmd/loop/main.go

```diff
@@ -34,6 +38,22 @@ var (
 	// maxMsgRecvSize is the largest message our client will receive. We
 	// set this to 200MiB atm.
 	maxMsgRecvSize = grpc.MaxCallRecvMsgSize(1 * 1024 * 1024 * 200)
+
+	// defaultMacaroonTimeout is the default macaroon timeout in seconds
+	// that we set when sending it over the line.
+	defaultMacaroonTimeout int64 = 60
+
+	tlsCertFlag = cli.StringFlag{
+		Name: "tlscertpath",
+		Usage: "path to loop's TLS certificate, only " +
+			"needed if loop runs in the same process " +
+			"as lnd (GrUB)",
+	}
+	macaroonPathFlag = cli.StringFlag{
+		Name: "macaroonpath",
+		Usage: "path to macaroon file, only needed if loop runs " +
+			"in the same process as lnd (GrUB)",
+	}
```

Comment:
If you are going to move the permissions from the grub pr to here, it also makes sense to require them for stand alone use

Reply:
Yes, I think when introducing macaroons to loop, it would make sense to also export the required permissions. But that's also not in the scope of this PR.

Reply:
Isn't it double work to first introduce the loop macaroons/permissions in grub and then later move them here? Also this 'only needed if loop runs in the same process' looks a bit strange to me, hinting towards an optimal order of implementation that is different

Reply:
Discussed offline, leaving this for later because it isn't trivial in grub to "merge" those macaroons. Only remove the reference to `GrUB`